content/source/docs/enterprise/upgrade/differences.html.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: enterprise2
+page_title: "Current vs. Legacy - Upgrading - Terraform Enterprise"
+sidebar_current: "docs-enterprise2-upgrading-differences"
+---
+
+# Differences Between Current and Legacy Terraform Enterprise
+
+Terraform Enterprise began as part of the discontinued Atlas suite, and today's version is a ground-up redesign. If you've previously used the legacy version of Terraform Enterprise, we hope you enjoy the new version's improvements.
+
+Several things are significantly different in the new Terraform Enterprise. This page is a brief summary of what to expect when upgrading from the legacy version.
+
+## Renamed or Moved Features
+
+### Workspaces Replace Environments
+
+Environments are now called [workspaces](../workspaces/index.html). Our goal in this change was to make it easier to on-board new Terraform users in your organization by avoiding an overloaded and ambiguous term.
+
+### Event Log is Now Audit Log (Private Installs Only)
+
+[audit]: ../private/logging.html#audit-logs
+
+Legacy Terraform Enterprise included an event log that logged details about run events. For the new Terraform Enterprise, we replaced it with a [comprehensive audit logging system][audit] that also logs all changes to organization and workspace settings.
+
+The new audit log is available on [private installs](../private/index.html) of Terraform Enterprise.
```

Comment:
It's actually running on SaaS TFE as well, though I'm not sure whether we want to mention that.

Reply:
I'd go with no! It's running... but it's not really _available_ to me as a user.

Reply:
Agreed. 

I wonder if its work linking to any docs (maybe `../private` has it) related to the new audit log, or clarifying: "other than the name change, the audit log contains the same information blah blah"

Reply:
The "comprehensive audit logging system" link goes to the audit log docs, but the link should probably be down in the second paragraph instead. 👍